server/common/stats/stats_helper.test.ts

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { SavedObjectsFindResponse } from '../../../../../src/core/server';
+import { MapSavedObjectAttributes } from '../../../common/map_saved_object_attributes';
+import { getStats } from './stats_helper';
+
+describe('getStats', () => {
```

Comment:
add a test case if someone called the stats api without any maps layers created.

Reply:
ack